web/src/beta/ui/fields/InputField/index.tsx

```diff
@@ -2,19 +2,47 @@ import { TextInput, TextInputProps } from "@reearth/beta/lib/reearth-ui";
 import CommonField, {
   CommonFieldProps
 } from "@reearth/beta/ui/fields/CommonField";
-import { FC } from "react";
-
+import { FC, useState, useEffect } from "react";
 
 export type InputFieldProps = CommonFieldProps &
   Pick<
     TextInputProps,
     "value" | "placeholder" | "onChange" | "onBlur" | "disabled" | "appearance"
   >;
 
-const InputField: FC<InputFieldProps> = ({ title, description, ...props }) => {
+const InputField: FC<InputFieldProps> = ({
+  title,
+  description,
+  value,
+  onChange,
+  onBlur,
```

Comment:
Since we added the logic of checking value, i feel `onBlur` is not a proper name now, could we rename it as `onChangeComplete` ? In this case
`onChange` -> trigger on every change
`onChangeComplete` -> trigger on blur and there do have changes
What do you think?
For fields we could remove `onBlur`, i guess no component wants blur exactly, all they need is chagneComplete, please let me know if you find any use case it needs `onBlur`, if so we can still have it.

Reply:
Same on other fields in this PR.